week1/index.js

```diff
@@ -1,25 +1,60 @@
-// To run this file: node index.js
-// Then in your browser: http://localhost:8080
 var http = require('http');
```

Comment:
Hi Eyad,

- Great indentation, easy to read
- Question to think about: when to use `const`, `var`, or `let`?